README.md

```diff
@@ -1 +1,49 @@
-# Hello World
+# Github Governance
+
+This readme is for building out our github gonvernance.
+
+## Top level SWRCB GitHub Documentation that covers our guiding principals, protocols, management decisions, and policies
+Also, this is open to discussion. 
+
+### Things that might go in this documnet or in this repo:
```

Comment:
This is a good time for brainstorming the general categories of stuff (as well as specifics; if you have an idea but don't see a section that it would fit into, propose a new one) we'd like to cover in our github governance.